src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -313,6 +313,11 @@ public static FormulaValue Upper(IRContext irContext, StringValue[] args)
             return new StringValue(irContext, args[0].Value.ToUpper());
         }
 
+        public static FormulaValue Proper(IRContext irContext, StringValue[] args)
+        {
+            return new StringValue(irContext, CultureInfo.CurrentCulture.TextInfo.ToTitleCase(args[0].Value));
```

Comment:
@MikeStall should `IRContext` expose the parameterized CultureInfo?

Reply:
It's available on EvalRunner. See Value() in LibraryText.cs as an example.

Reply:
I edited this and will resolve the comment. Let me know if you think this is not the correct way.